desktop/apps/auction/routes.js

```diff
@@ -37,22 +37,14 @@ export async function index (req, res, next) {
   const auctionArticles = new Articles(articles)
 
   try {
-    const html = makeTemplate('index.jade', {
-      basePath: res.app.get('views'),
-      locals: {
-        ...res.locals,
```

Comment:
It could be that Express reuses the `res` data structure between requests. For some reason, I think I recall something like that. Perhaps to save GC pressure or something like that?